File: C/Iniciante/2862/2862.c
#include <stdio.h>

int main() {

    int C, N, i;

    scanf("%d", &C);

    for (i = 0; i < C; i++) {
        scanf("%d", &N);

        if (N > 8000) {
            printf("Mais de 8000!\n");
        } else {
            printf("Inseto!\n");
        }
    }

    return 0;
}

File: Python/Iniciante/2862.py
# Problema 2862 - URI - Iniciante - Nível 1

# Lê a quantidade de valores
n = int(input())

for i in range(n):
    # Lê o número e verifica se é maior que 8000
    # Se for maior que 8000, ele imprime "Mais de 8000!"
    # Caso contrário, imprimirá "Inseto!"
    print('Mais de 8000!' if int(input()) > 8000 else "Inseto!")